jdaviz/configs/imviz/plugins/links_control/links_control.py

```diff
@@ -49,6 +90,17 @@ def __init__(self, *args, **kwargs):
                                                selected='link_type_selected',
                                                manual_options=['Pixels', 'WCS'])
 
+        self.viewer = ViewerSelect(
+            self, 'viewer_items', 'viewer_selected', 'multiselect'
+        )
+        self.layer = LayerSelect(
+            self, 'layer_items', 'layer_selected', 'viewer_selected',
+            'multiselect', only_wcs_layers=True
+        )
+        self.orientation_layer_label = AutoTextField(
+            self, 'new_layer_label', 'new_layer_label_default', 'new_layer_label_auto', None
+        )

```

Comment:
this is really close to using the `AddResultsMixin`, except that you want to apply the results to _all_ viewers, right?  I wonder if we might want to consider writing that logic upstream if there would ever be another use for that (and might save you from needing the `_add_data_to_all_viewers` method)?

Reply:
I agree that this is an unimplemented special case for `AddResultsMixin`. Should I make a follow-up tech debt ticket to do that change?

Reply:
that works for me, thanks!